tests/steps/build_commands.py

```diff
@@ -75,11 +75,16 @@ def step_impl(context, target, option):
 
 @when("the draft is broken")
 def step_impl(context):
+    import platform
+    if any(platform.system() == p for p in ['Darwin', 'BSD']):
```

Comment:
I think that this works too.

```suggestion
    if platform.system() in ['Darwin', 'BSD']:
```

Reply:
Ah right, yeah I was using `any()` because I was initially using `p in platform.system()` instead of `platform.system() == p` but this is simpler

Reply:
Oh and thinking about this more, it returns `FreeBSD` on that platform, so I switched back to any with a substring match